docs/pages/setup/reference/networking.mdx

```diff
@@ -18,22 +22,51 @@ following use cases:
 
 - You have multiple identical services, like proxies, behind a load balancer.
 - You want Teleport to issue an SSH certificate for the service with the additional
-  principals, e.g.host names.
+  principals, e.g., host names.
+</TabItem>
+<TabItem scope={["cloud"]} label="Teleport Cloud">
+
+Teleport Nodes have an optional `public_addr` property that you can modify in
+each service's configuration file. The public address can take an IP or a DNS
+name. It can also be a list of values:
+
+```yaml
+public_addr: ["proxy-one.example.com", "proxy-two.example.com"]
+```
+
+Specifying a public address for a Teleport Node may be useful in the
+following use cases:
+
+- You have multiple identical services, like proxies, behind a load balancer.
+- You want Teleport to issue an SSH certificate for the service with the additional
+  principals, e.g., host names.
+</TabItem>
+</Tabs>
+
 
 ## HTTP CONNECT proxies
 
 Some networks funnel all connections through a proxy server where they can be
-audited and access control rules are applied. For these scenarios, Teleport supports HTTP CONNECT tunneling.
+audited and access control rules can be applied. For these scenarios, Teleport
+supports HTTP CONNECT tunneling.
+
+To use HTTP CONNECT tunneling, simply set either the `HTTPS_PROXY` or
+`HTTP_PROXY` variables of the `teleport` daemon's shell environment. When the
+daemon builds and establishes a reverse tunnel to the main cluster, it will
+funnel all traffic through the selected proxy. 
 
-To use HTTP CONNECT tunneling, simply set either the `HTTPS_PROXY` or `HTTP_PROXY` environment variables and when Teleport builds and establishes the reverse tunnel to the main cluster, it will funnel all traffic through the proxy. Specifically, if using the default configuration, Teleport will tunnel ports `3024` (SSH, reverse tunnel) and `3080` (HTTPS, establishing trust) through the proxy.
+If using the default configuration, Teleport will tunnel ports `3024` (SSH,
+reverse tunnel) and `3080` (HTTPS, establishing trust) through the proxy.
```

Comment:
I'm not sure what we're trying to say about port 3080 here. It is not "tunneled through the proxy" - it's just the listen port on the proxy. There is no tunneling taking place on port 3080.